crates/viewer/re_dataframe_ui/src/display_record_batch.rs

```diff
@@ -54,21 +51,19 @@ enum ComponentData {
         dict: ArrowInt32DictionaryArray,
         values: ArrowListArray,
     },
+    Scalar(ArrowArrayRef),
```

Comment:
Do not worry too much about this change and anything with `ComponentData`. This thing will be removed in favour of the future `ArrayCell` thing.